### PR TITLE
[Main Course] - GA intro

### LINKED ISSUE #27: GA Intro
# Outline

- What is GA
- What can you do with GA
- Set up with GA
- Insight
- Conclusion

# Milestone
- 7/31 - Outline complete
- 8/1 - Draft complete
- 8/1 - Review complete
- date - Release

### PR DESCRIPTION
**Pull request by Realvincentyuan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #27  (GA intro)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+# 浅谈Google Analytics
+
+!!! info
+    作者：袁子弹起飞，发布于2021-07-31，阅读时间：约6分钟，微信公众号文章链接：[:fontawesome-solid-link:](https://mp.weixin.qq.com/s/p_Cva92Md_N7k6ohXzc-fA)
+
+## 1 前言
+
+现今互联网上的很多产品、战略决策都由数据驱动，以BulletTech为例，在运营微信公众时，通过后台数据我们对每篇文章都会进行流量来源、裂变和阅读完关注等重要指标的监控，这些数据能够帮助我们分析读者的偏好、流量渠道的优劣，从而灵活地调整文章主题和宣传策略。对于已经有良好数据分析基础设施的平台，现成的解决方案能省去决策者很多时间，但如果自己从头建了一个产品，如何构建数据分析的基础设施和策略呢？这篇文章将以BulletTech的博客为例，向大家介绍如何使用Google Analytics（GA）分析网站表现。
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md:12]: 运营微信公众号时

File: docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## 2 Google Analytics的作用
+
+GA是Google发布的一款用于网站和移动应用分析的免费工具，将GA配置在网站上后，它能够抓取到网站上的用户行为数据，帮助你优化网站。GA在如下几个方面的分析中起到了重要作用：
+
+- 市场推广：GA能够清楚地展示各个渠道用户在网站上的行为及转化率，帮助优化针对劣势渠道的优化
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md:18]: 帮助提升针对劣势渠道的优化 会不会更好一点

**Review by jhqian0909 — APPROVED** [review]

File: docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md (hunk 2 above)
Comment (Realvincentyuan) [reply docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md]: > 帮助提升针对劣势渠道的优化 会不会更好一点

已修复 ：）

File: docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md (hunk 1 above)
Comment (Realvincentyuan) [reply docs/Main_Course/Data_Analytics/2021-07-31-GA-Intro.md]: 已修复

**Pull request merged**